app/models/target.rb

```diff
@@ -34,8 +34,6 @@ def get_progress
         conference.registrations.where('created_at < ?', due_date).count
       when Target.units[:program_minutes]
         conference.current_program_minutes
-      else
-        0
```

Comment:
@ChrisBr I think it would be a better practice to keep the `else`. In case something goes wrong with our validations, we won't end up with nil value, but we will have 0 instead.